assets/prometheus-k8s/prometheus.yaml

```diff
@@ -151,6 +151,7 @@ spec:
   serviceMonitorSelector: {}
   thanos:
     baseImage: quay.io/openshift/origin-thanos
+    image: quay.io/thanos/thanos:v0.10.0
```

Comment:
is this correct @s-urbaniak ?

Reply:
this is overriden at runtime in https://github.com/openshift/cluster-monitoring-operator/blob/8a08a5422f45bc2393a97258e5b6e11b52aaa32b/pkg/manifests/manifests.go#L1223-L1225